exonum-java-binding/core/rust/Cargo.toml

```diff
@@ -57,3 +57,5 @@ exonum-derive = { git = "https://github.com/exonum/exonum", tag = "v0.13.0-rc.2"
 exonum-cli = { git = "https://github.com/exonum/exonum", tag = "v0.13.0-rc.2" }
 exonum-proto = { git = "https://github.com/exonum/exonum", tag = "v0.13.0-rc.2" }
 exonum-supervisor = { git = "https://github.com/exonum/exonum", tag = "v0.13.0-rc.2" }
+exonum-crypto = { git = "https://github.com/exonum/exonum", tag = "v0.13.0-rc.2" }
```

Comment:
Those two lines are required to make anchoring happy in case we don't rely on published exonum dependencies.